pkg/virt-controller/watch/migration/migration.go

```diff
@@ -413,16 +415,28 @@ func (c *Controller) canMigrateVMI(migration *virtv1.VirtualMachineInstanceMigra
 }
 
 func (c *Controller) failMigration(migration *virtv1.VirtualMachineInstanceMigration) error {
-	err := backendstorage.MigrationAbort(c.clientset, migration)
+	_, err := backendstorage.MigrationAbort(c.clientset, migration)
 	if err != nil {
 		return err
 	}
 	migration.Status.Phase = virtv1.MigrationFailed
+
 	return nil
 }
 
-func (c *Controller) updateStatus(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance, pods []*k8sv1.Pod, syncError error) error {
+func (c *Controller) interruptMigration(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
I think you are missing to set the migration as `Completed`  and the timestamp in the VMI object.

Reply:
Whether the migration is completed or not is debatable... Do you think it would make more sense to set it as such?

Reply:
We can leave it like this, but the vmi.IsMigrating returns true. 
When I was working on https://github.com/kubevirt/kubevirt/pull/13636,I noticed that the printableState was `Migrating` instead of `Stopped`. It was a bit weird from the user perspective that that they need to restart the VM in that case. That's why I moved the isStopped earlier in https://github.com/kubevirt/kubevirt/pull/13636/commits/98f10eeddd3a7335b8b193fc6babedc4c1508730. But as you said, it is debatable.